ESP_ATMod/command.cpp

```diff
@@ -1643,6 +1669,165 @@ void cmd_AT_CIPMUX()
 	}
 }
 
+/*
+ * AT+CIPCIPSERVER - Deletes/Creates TCP Server
+ */
+void cmd_AT_CIPSERVER()
+{
+
+	if (!gsCipMux)
+	{
+		Serial.printf_P(MSG_ERROR);
+		return;
+	}
+	uint8_t error = 1; // 1 = generic error, 0 = ok
+	uint16_t offset = strlen("AT+CIPSERVER");
+
+	bool stop = false;
+	uint32_t port = 0;
+
+	do
+	{
+		if (inputBuffer[offset] != '=')
+			break;
+
+		++offset;
+		stop = inputBuffer[offset] == '0';
+		++offset;
+
+		if (inputBufferCnt > offset)

```

Comment:
I think it should be offset + 2, there is a CR/LF at the end of the buffer

Reply:
next character can my a `,` if there is a second parameter (port)

Reply:
Juraj, it really doesn't work. Try AT+CIPSERVER=1, it gives error

Reply:
it should give ERROR. syntax is `AT+CIPSERVER=<mode>[,<port>]`

Reply:
Sorry, I hate arguing but it should not. I tested it on AT command processor for ESP32-C3. The command `AT+CIPSERVER=1` is perfectly valid and opens port 333.

Reply:
yes `AT+CIPSERVER=1` is valid, `AT+CIPSERVER=1,` is not. byt please don't check with AT2. the syntax is not the same on some commands.  I expected `inputBuffer` to be without CRLF.

Reply:
I see the misunderstanding now, sorry. I should have marked the text in my contribution above. I agree with your last reply.